modules/mockk/src/commonMain/kotlin/io/mockk/NonMockableClassesTest.kt

```diff
@@ -0,0 +1,59 @@
+package io.mockk
+
+
+
+import kotlin.test.Test
+
+import kotlin.test.assertFailsWith
+
+
+
+class NonMockableClassesTest {
+
+
+
+  @Test
+
+  fun `should throw exception when mocking System class`() {
```

Comment:
I think you can make this test parameterized and pass the list of non mockable classes.

Reply:
Can you Help me with it ?